src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  max-width: 100%;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.logo {
+  padding-top: 5px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.nav {
+  display: flex;
+  align-items: stretch;
+}
+
+.nav__section {
+  list-style: none;
+  display: flex;
+  margin-left: 20px;
+}
+
+.nav__sectionApple--margin--0 {
+  list-style: none;
+  display: flex;
+  margin-left: 0;
+  margin-right: 0;
+}
```

Comment:
why do you need the specific styles for sectionApple?

Reply:
I had a mistake and I so to fix it

Reply:
You should not add additional class to first element if you need to specify some rules
You can use :first-child instead
https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child